sorc/build.mpas

```diff
@@ -26,7 +26,7 @@ module load rrfs/${MACHINE}.intel
 module list
 
 cd ${HOMErrfs}/sorc/MPAS-Model
-cp ../_workaround_/MPAS-Model/Registry.xml src/core_atmosphere
+cp ../_workaround_/Registry.xml src/core_atmosphere/Registry.xml
```

Comment:
Could you use what's in current build.mpas?
```
cp ../_workaround_/MPAS-Model/Registry.xml src/core_atmosphere
```
This way, we don't have a change history for this file on this PR. Thanks!